packages/core/package-manager/src/index.js

```diff
@@ -5,3 +5,4 @@ export * from './Pnpm';
 export * from './Yarn';
 export * from './MockPackageInstaller';
 export * from './NodePackageManager';
+export * from './installPackage';
```

Comment:
Maybe we can avoid exporting this from the package root and just import from the subpath directly in the test? This shouldn't really be public...

Reply:
Done.